src/main/java/com/quirko/gui/GuiController.java

```diff
@@ -55,6 +58,9 @@
     @FXML
     private GameOverPanel gameOverPanel;
 
+    @FXML
+    private VERY_SPECIAL_Panel VERYSPECIAL_Panel;

```

Comment:
Variable naming is not consistent with the rest of the code: 

```VERY_SPECIAL_Panel``` => ```VerySpecialPanel```

Reply:
Düzelttim.